packages/lib-core/src/store/PluginStoreContext.tsx

```diff
@@ -31,3 +31,19 @@ export const usePluginStore = (): PluginConsumer & PluginManager => {
 
   return store;
 };
+
+/**
+ * React hook that provides access to a feature flag.
+ *
+ * @example
+ * ```ts
+ * const [flag, setFlag] = useFeatureFlag('MY_FLAG');
+ * ...
+ * setFlag(true);
+ * ```
+ */
+export const useFeatureFlag = (flagName: string): [boolean, (newValue: boolean) => void] => {
+  const store = usePluginStore();
```

Comment:
```suggestion
  const pluginStore = usePluginStore();
```

I'd generally prefer `pluginStore` since `store` might be too ambiguous.